include/openenclave/host.h

```diff
@@ -384,6 +384,25 @@ void oe_free_key(
     uint8_t* key_info,
     size_t key_info_size);
 
+/**
+ * oe_verify_tls_cert
```

Comment:
It's not obvious that this is not general support for TLS certs; even in the enclave we expect that they will want to auth against non-enclave servers and so will need deal with normal TLS cert approval.

We might want to consider qualifying this as an `oe_verify_enclave_tls_cert` or something with a special term indicating our unique cert format e.g. `oe_verify_attestation_cert`

Reply:
Thanks for the great feedback.

I have renamed them to the following :
oe_generate_attestation_cert
oe_free_attestation_cert
oe_verify_attestation_cert